third_party/nvidia/hopper/lib/Transforms/WarpSpecialization/WSCodePartition.cpp

```diff
@@ -1404,6 +1422,14 @@ void doCodePartition(triton::FuncOp &funcOp, unsigned numBuffers) {
   groupChannels(channels, channelsGroupedByProducers,
                 channelsGroupedByConsumers, orderedChannels);
 
+  // Step 3: Create buffers. An array of buffers for each channel.
```

Comment:
Nit: Can you relabel the total step counts? We have multiple step 3s.

Reply:
Let me do it. I was thinking about that but `reorderProducerOps` will probably go away.